eng/pipelines/client.yml

```diff
@@ -13,5 +13,5 @@ pr: none
 jobs:
 - template: templates/jobs/archetype-sdk-client.yml
   parameters:
-    ProjectFile: $(ProjectFile)
+    ServiceDirectory: '*'
```

Comment:
I think it should be set to `*` in the web ui variables, in this file it should just be set to `$(ServiceDirectory)`